Plugins/MeshFeatureTracking/src/MeshFeatureTrackingComponent.cpp

```diff
@@ -57,7 +57,8 @@ int MeshFeatureTrackingComponent::getMaxV() const {
             ->getRenderObject( m_pickedRoIdx )
             ->getMesh()
             ->getGeometry()
-            .m_vertices.size();
+            .vertices()
+            .size();
     }
     return 0;
 }
```

Comment:
I don't like so much this kind of notation that call getters through pointers in cascade. What if on pointer is null somewhere ? why is the guard "if ( m_data.m_mode != PickingMode::RO && getRoMgr()->exists( m_pickedRoIdx ) )" sufficient ?

Reply:
Well, I do agree.
The problem is that this plugin has been coded this way, and we didn’t try to fix such style issues.
I can add guard if really needed, but in any case this need to be fixed at many places in Radium source code, which is out of the scope of this PR.